Don't call 'make' blindly and use the correct GNU make program.
Also don't discard the output of the make program so that one can figure out
errors or see some progress.

--- bin/rosmake~	2010-06-23 02:35:54.000000000 +0200
+++ bin/rosmake	2011-01-16 00:55:24.000000000 +0100
@@ -41,8 +41,9 @@
         return True
 
     #cmd = ["make", "-C", os.path.join(os.environ["ROS_ROOT"], "tools/rospack")] Doesn't work with cmake 2.6, will trigger rebuilds due to different paths.  Fixed in 2.8.
-    cmd = ["bash", "-c", "cd %s && make"%(os.path.join(os.environ["ROS_ROOT"], "tools/rospack"))]
-    if subprocess.call(cmd, stdout=subprocess.PIPE, stderr=subprocess.PIPE) :
+    cmd = ["bash", "-c", "cd %s && @MAKE@"%(os.path.join(os.environ["ROS_ROOT"], "tools/rospack"))]
+    print "Executing", cmd
+    if subprocess.call(cmd) :
         print "Rospack failed to build" 
         return False
 
